sql-statements/sql-statement-alter-table.md

```diff
@@ -154,7 +154,7 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-- 不支持在单个 `ALTER TABLE` 语句中进行多个更改。
+- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。
```

Comment:
tidb check consistency with the schema before ddl. if some schema is added in the ddl, can't alter it.eg.
`ALTER TABLE ADD INDEX i(b), DROP INDEX i;